libs/core-ui/src/lib/Cohort/Cohort.ts

```diff
@@ -10,6 +10,12 @@ import {
 import { JointDataset } from "../util/JointDataset";
 import { ModelExplanationUtils } from "../util/ModelExplanationUtils";
 
+export enum ErrorDetectorCohortSource {
```

Comment:
since we are moving cohort to shared place, we should get rid of word "error"

Reply:
Just `CohortSource` should be sufficient.

More generally, there is `Cohort` in interpret and `ErrorCohort` which is used in error-analysis and model-assessment. That won't be changeable easily.